vm-to-vm-bandwidth-meter/azuredeploy.json

```diff
@@ -0,0 +1,611 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "probeVmSize": {
```

Comment:
Ok, but then let's reduce the list to say one per series.  We're adding a 150 lines to the template for this and users are more likely to fail matching the region/series than they are to struggle to figure out how to change a variable value...